src/nodeMatchers.ts

```diff
@@ -216,3 +51,10 @@ export function cascadingMatcher(...matchers: NodeMatcher[]): NodeMatcher {
     return null;
```

Comment:
While we're here, maybe `cascadingMatcher` isn't the best name?  Maybe something like `unionMatcher`?  `matchAny`?  Idk.  But I had to scratch my head to remember what it did and I'm the one that named it in the first place 😅

Reply:
I didn't have trouble with the name, but `matchAny` would probably do just as well. I liked cascading because it implied that the matchers would be tried in order.

Reply:
Yeah that's fair.  I guess cascading has precedent with "cascading style sheets".  Happy to leave it if you find it clear.  problem with `matchAny` is that it wouldn't play nicely if we align on having `fooMatcher` as convention